packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts

```diff
@@ -0,0 +1,54 @@
+import { html, when, slotted } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "./breadcrumb-item";
+import { Breadcrumb } from "../breadcrumb";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(BreadcrumbItem:class)} component.
+ * @public
+ */
+export const BreadcrumbItemTemplate = html<BreadcrumbItem>`
+    <div role="listitem" class="listitem" part="listitem">
+        ${startTemplate}
+        <slot
+            ${slotted({
+                property: "defaultSlottedNodes",
+                filter: value =>
+                    (value.nodeType === 3 && value.textContent!.trim().length !== 0) ||
+                    value.nodeType === 1 ||
+                    false,
+            })}
+        ></slot>
+        <slot name="control">
+            ${when(
+                x => x.defaultSlottedNodes.length === 0,
+                html<BreadcrumbItem>`
+                    <a
+                        class="control"
+                        part="control"
+                        href="${x => x.href}"
+                        aria-current="${x => (x.isCurrent ? "page" : void 0)}"
+                    >
```

Comment:
What happens if I want to pass additional attributes here? Should they pass to the host or to this? Should we delegate focus here?

Reply:
yeah, delegate focus should be here. Will add that.

Reply:
added `delegatesFocus` and aria attributes to the anchor